README.md

```diff
@@ -271,6 +271,31 @@ As of Bazel 0.26 this feature is still experimental, so also add this line to th
 common --experimental_allow_incremental_repository_updates
 ```
 
+#### @bazel/hide-bazel-files
+
+We recommend adding the `@bazel/hide-bazel-files` utility as a postinstall step to any `package.json` files that
```

Comment:
If we recommended this then we should add it to the `create` package so that new projects get it by default

Reply:
👍